CLDR-18616 Update Info Hub following May 17 push to production (#4716)
unicode-org/cldr: docs/site/translation.md

💡🆕

The sequence above marks items that have been recently added. In your browser you can copy this sequence, then use ⌘-F (Mac) or Ctrl-F (Windows) to find all the places it occurs.
- 💡🆕 Added missing date & time formats:
  - GyM and GyMEd
  - Time formats: EBh, Eh, EH when there is an existing EBhm, Ehm, and EHm in the calendar respectively.

>   - Changed HH patterns in available and internval formats to include a reference to hour since seeing an hour number alone is ambigious.

Reviewer: ```suggestion
  - Changed HH patterns in available and internal formats to include a reference to hour since seeing an hour number alone is ambigious.
```

> - 💡🆕 Sark, CQ, is now in modern coverage

Reviewer: ```suggestion
- 💡🆕 The region Sark, CQ, is now in modern coverage
```

- New flexible format and interval patterns
- Gregorian Calendar (Year First) calendar
- Enhanced "Show Hidden"

When a section below changes, the date will be in the header.

### Starting Submission

> During Submission, please read the [CLDR training](#cldr-training-for-new-linguists) (if new to the survey tool), please focus on the missing, provisional, and errors.

Reviewer: ```suggestion
Before you start Submission, please read the [CLDR training](#cldr-training-for-new-linguists) (if new to the survey tool). Please prioritize the sections Missing, Provisional, and Errors.
```

in English that would produce “tomorrow, 3:00 PM”. If instead you want the same combining behavior for a relative date with a single time as for a
fvfixed date with single time (as was the case in CLDR 47 and earlier), then for each length style copy the existing “atTime” form to the new “relative” form.

#### Missing date & time patterns

> 💡🆕 Currently dates and times are sometimes poorly formatted or ambiguous due to missing patterns. The additional patterns have been added to resolve this issue:

Reviewer: ```suggestion
💡🆕 Some dates and times are ambiguous due to missing patterns. These additional patterns have been added to resolve this issue:
```

  - GyM and GyMEd

>   - Time formats: EBh, Eh, EH when there is an existing EBhm, Ehm, and EHm in the calendar respectively.

Reviewer: ```suggestion
  - Time formats: EBh, Eh, EH when there is an existing EBhm, Ehm, and EHm in the respective calendar.
```

>   - Changed HH patterns in available and internval formats to include a reference to hour since seeing an hour number alone is ambigious.

Reviewer: ```suggestion
  - HH patterns in available and internal formats have been changed to include a reference to hour since seeing an hour number alone is ambigious.
```
